NonEmptyList/fromList

```diff
@@ -0,0 +1,16 @@
+  λ(a : Type)
```

Comment:
Can we add the following documentation, feel free to reword or change.


{-
Create a NonEmptyList from a List as long as there is at least one element in the List. Otherwise return None.

Examples:

```
./NonEmptyList/fromList Natural [1, 2, 3, 4, 5]
= Some { head = 1, tail = [ 2, 3, 4, 5 ] }

./NonEmptyList/fromList Natural ([] : List Natural)
= None { head : Natural, tail : List Natural }
```
-}